bitsail-base/src/main/java/com/bytedance/bitsail/base/metrics/reporter/impl/PrometheusPushGatewayMetricReporterBuilder.java

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
How to test the final effect

Reply:
Now I just use hadoop source to test locally, next i will add test unit and document.